src/dirs.proj

```diff
@@ -7,11 +7,20 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Project Include="ref.builds" />
     <Project Include="src.builds" />
-    <Project Include="shims\shims.proj" />
   </ItemGroup>
 
+  <Target Name="BuildRefAndShims" BeforeTargets="BuildAllProjects">
```

Comment:
I would prefer not to do a direct msbuild on these projects lets try and make sure we let the traversal targets do its job. If the order isn't holding for some reason we should figure out why.